### ISSUE TITLE
should we add a progress bar?

### ISSUE THREAD

**Issue by fmichonneau** [iDigBio/ridigbio#10]
Should we add a progress bar to monitor status of larger requests?

**Comment by mjcollin** [comment]
Is there a way to tell if the query is being run interactively vs as part of a script? Or direct output someplace that gets eaten when there is no console?

**Comment by fmichonneau** [comment]
> Is there a way to tell if the query is being run interactively vs as part of a script? 

Yes, `interactive()` returns TRUE when R is being run interactively.

We should probably use: https://github.com/gaborcsardi/progress

**Comment by mjcollin** [comment]
Looks like a nice package. I think this is a nicety now but we definitely will need to do this when the download API endpoints get added. ETA on that is July-ish after the next portal and indexes get rolled out.
